auth_signup_verify_email/controllers/main.py

```diff
@@ -49,7 +53,22 @@ def passwordless_signup(self, values):
             # Agnostic message for security
             qcontext["error"] = _(
                 "Something went wrong, please try again later or contact us.")
-            return http.request.render("auth_signup.signup", qcontext)
+            return request.render("auth_signup.signup", qcontext)
 
         qcontext["message"] = _("Check your email to activate your account!")
-        return http.request.render("auth_signup.reset_password", qcontext)
+        return request.render("auth_signup.reset_password", qcontext)
+
+    def _get_user_lang(self):
+        """Retrieve current user language."""
+        langs = request.env['res.lang'].search_read([], ['code'])
+        supported_langs = [
+            lang['code'] for lang in langs
+        ]
+        lang = 'en_US'  # fallback
+        if hasattr(request, 'website'):
+            # default to site lang
+            lang = request.website.default_lang_code
+        if request.lang in supported_langs:
```

Comment:
Is it even possible that a request that comes from the website has a lang that is not among the langs accepted by the website? 🤔  I think that in such case, Odoo will redirect the request to the website's default language and then process it. In such case, all of this method makes no sense and you can just use `request.lang`.

Reply:
hmm, I remember I've done this for a good reason but I was so idiot to not put proper comments :sob: 
Nevertheless, I did this on v9 and years ago, so: let's drop it. I can put it back if needed later on :+1: